analyses/molecular-subtyping-EPN/02_ependymoma_generate_all_data.py

```diff
@@ -0,0 +1,179 @@
+            YAP1_MAMLD1 = fusion.get_value(line[3], "YAP1--MAMLD1")
+            YAP1_FAM118B = fusion.get_value(line[3], "YAP1--FAM118B")
+            C11orf95_MAML2 = fusion.get_value(line[3], "C11orf95--MAML2")
+            # Looking for expression data and creating Z-scores  
+            RELA_zscore = (fpkm_df.get_value("RELA", line[3])-RELA_mean)/RELA_stdev
+            L1CAM_zscore = (fpkm_df.get_value("L1CAM", line[3])-L1CAM_mean)/L1CAM_stdev
+            ARL4D_zscore = (fpkm_df.get_value("ARL4D", line[3])-ARL4D_mean)/ARL4D_stdev
+            CLDN1_zscore = (fpkm_df.get_value("CLDN1", line[3])-CLDN1_mean)/CLDN1_stdev
+            CXorf67_zscore = (fpkm_df.get_value("CXorf67", line[3])-CXorf67_mean)/CXorf67_stdev
+            TKTL1_zscore = (fpkm_df.get_value("TKTL1", line[3])-TKTL1_mean)/TKTL1_stdev
+            GPBP1_zscore = (fpkm_df.get_value("GPBP1", line[3])-GPBP1_mean)/GPBP1_stdev
+            IFT46_zscore = (fpkm_df.get_value("IFT46", line[3])-IFT46_mean)/IFT46_stdev
+            # Looking for chromosomal CNA based on DNA BSID's. Some of the corresponding DNA ID's show "NA", so splitting with if statement
+            if line[2]!="NA":
+                oneqvalue = (CNA.get_value("1q",line[2]))
+                nineploss = (CNA.get_value("9p",line[2]))
+                nineqloss = (CNA.get_value("9q",line[2]))
+                sixploss = (CNA.get_value("6p",line[2]))
+                sixqloss = (CNA.get_value("6q",line[2]))
+                elevenq = (CNA.get_value("11q",line[2]))
+                breaks_density =  breakpoint_density.get_value(line[2], "breaks_density")
+                if int(oneqvalue) < 0:
+                    line.append("1")
+                else:
+                    line.append("0")
+                if int(nineploss) < 0:
+                    line.append("1")
+                else:
+                    line.append("0")    
+                if int(nineqloss) < 0:
+                    line.append("1")
+                else:
+                    line.append("0")    
+                if int(sixploss) < 0 :
+                    line.append("1")
+                else:
+                    line.append("0")    
+                if int(sixqloss) < 0:
+                    line.append("1")
+                else:
+                    line.append("0")
+                if int(elevenq) < 0:
+                    line.append("1")
+                else:
+                    line.append("0")
+                if int(elevenq) > 0:
+                    line.append("1")
+                else:
+                    line.append("0")
+                # Adding all the data collected so far for every sample with RNA and DNA BSID to "line"     
+                line.extend((nfkb_gsva_score, C11orf95_RELA, LTBP3_RELA, PTEN_TAS2R1, C11orf95_YAP1, YAP1_MAMLD1, YAP1_FAM118B, C11orf95_MAML2, breaks_density, RELA_zscore, L1CAM_zscore,ARL4D_zscore, CLDN1_zscore, CXorf67_zscore, TKTL1_zscore, GPBP1_zscore, IFT46_zscore))
```

Comment:
You may want to break up the contents of the extend here into logical groups that you can separate onto their own lines.
Something like the below, just to keep the line lengths in check and improve readability. If you do the same with the headers it will be easier to figure out if you accidentally change the order and the header and contents get out of sync.
```suggestion
                line.extend((nfkb_gsva_score, C11orf95_RELA, LTBP3_RELA, PTEN_TAS2R1, 
                             C11orf95_YAP1, YAP1_MAMLD1, YAP1_FAM118B, C11orf95_MAML2, breaks_density,
                             RELA_zscore, L1CAM_zscore, ARL4D_zscore, CLDN1_zscore, 
                             CXorf67_zscore, TKTL1_zscore, GPBP1_zscore, IFT46_zscore))
```